examples/ICESat-2_DAAC_DataAccess_Example.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    print(region_a.dataset)
</pre><p>when i execute print(region_a.product), the error ocurs. then i modify the code to print(region_a.dataset), the code successfully execute.  i have update the icepyx to the  latest version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>